specification/loadtestservice/LoadTestService/models.tsp

```diff
@@ -508,7 +508,14 @@ model RegionalConfiguration {
   @doc("""
       The number of engine instances to execute load test in specified region. Supported values are in range of 1-400.
     """)
-  engineInstances: int32;
+  @madeOptional(APIVersions.v2025_03_01_preview)
+  engineInstances?: int32;
+
+  @doc("""
+      The percentage of load to be distributed in specified region. Supported values are in range of 1-100.
+    """)
+  @added(APIVersions.v2025_03_01_preview)
```

Comment:
Lets use `@minValue` and `@maxValue` decorators to make the spec more tight. [Reference](https://typespec.io/docs/standard-library/built-in-decorators/#@minValue)

Reply:
Done